src/components/feedback/RatingStar.tsx

```diff
@@ -19,8 +27,8 @@ export const RatingStar: FC<RatingStarProps> = ({ onChange, ratingValue, current
         onChange={onChange}
         className={styles['rating-input']}
       />
-      <label htmlFor={`rating${ratingValue}`} className={styles['rating-label']}>
-        {currentRating >= ratingValue ? <IoMdStar size={30} /> : <IoMdStarOutline size={30} />}
+      <label htmlFor={`rating${ratingValue}`} className={styles[cssClassName!]}>
```

Comment:
no need to create enum for star className and incapsulate custom style inside this component, you can just use className={className} and define additional styles in parent component